routing/cross_mwm_connector_serialization.hpp

```diff
@@ -47,14 +75,29 @@ class CrossMwmConnectorSerializer final
     }
 
     template <class Sink>
-    void Serialize(serial::CodingParams const & codingParams, uint32_t bitsPerOsmId,
+    void WriteCrossMwmId(connector::OsmId const & id, uint8_t bits, BitWriter<Sink> & w) const
+    {
+      w.WriteAtMost64Bits(id.Get(), bits);
+    }
+
+    template <class Sink>
+    void WriteCrossMwmId(connector::TransitId const & id, uint8_t bitsPerCrossMwmId, BitWriter<Sink> & w) const
+    {
+      CHECK_EQUAL(bitsPerCrossMwmId, connector::CalcBitsPerTransitId(), ("Wrong TransitId size."));
+      w.WriteAtMost64Bits(id.m_stop1Id, connector::kStopIdBits);
```

Comment:
It would be nice to assert here that kStopIdBits <= 64. And that kLineIdBits <= 32.

Reply:
Is not it enough to have this code at the end of the file?
```
static_assert(connector::kOsmIdBits == 64, "Wrong kOsmIdBits.");
static_assert(connector::kStopIdBits == 64, "Wrong kStopIdBits.");
static_assert(connector::kLineIdBits == 32, "Wrong kLineIdBits.");
```

Reply:
OK